platform/product-management/release-plan-template.md

```diff
@@ -6,6 +6,38 @@ So: **how might we craft a release plan to test our product "in the wild" at a s
 
 That's what this Release Plan Template is for! And note - there are feature toggles and beta banners at your disposal that you can use as a part of your plan.
 
+## Do I need unmoderated production testing?
```

Comment:
I suggest changing the format of this doc so that this section is paired with the "unmoderated production testing planning" section. I could see it working either as: 

Intro
PLANNING SECTION
Planning - moderated
Planning - unmoderated - do I need to do unmoderated prod testing?
Planning - unmoderated - if yes, details
Planning - phased/staged rollout (need to decide term, +1 andrea's comments) 
RESULTS SECTION
Results - moderated
Results - unmoderated
Results - phased/staged rollout
POST-LAUNCH CHECK IN Qs

OR ... 

Intro
Planning - moderated
Results - moderated
Planning - unmoderated - do I need to do unmoderated prod testing?
Planning - unmoderated - if yes, details
Results - unmoderated
Planning - phased/staged rollout
Results - phased/staged rollout 
POST-LAUNCH CHECK IN Qs

Personally I think the first option here might make more sense, but that's not how we have it now. 

Anyway another thought is — do we need/want heuristics for "do I need to do moderated production testing" as well? Are they the same heuristics? Should we ask Emily W or Kevin H from a UX perspective?

Reply:
I think the part I'm having a hard time with is that, as a _template_, it feels a little weird to mix in the "do I need this section or not?" questioning + reasoning with pieces we expect the VFS teams to fill out. I can put it in the middle but I think that makes it more awkward. (trying it with this revision to compare though)

re: "do we want/need heuristics for 'do I need to do moderated production testing' as well", I'd rather lean keeping this simpler and coming back to add that if we feel there's a reason for it later.

Reply:
Taking a look at the doc formatting, I like it better this way (the way you've changed it to — with the heuristics mid-doc). I hear you on awkwardness, but I think this is less awkward than having it at the top (_unless_ we also have "Do I need moderated production testing" at the top, which we don't). 

+1 on not addressing moderated production testing heuristics for now. I think we do need them, but feel good saying that's out of scope and coming back to it later.